src/install/mod.rs

```diff
@@ -151,7 +151,7 @@ fn prebuilt_url(tool_name: &str, owner: &str, version: &str) -> Option<String> {
         };
 
         let url = format!(
-            "https://github.com/{0}/{1}/releases/download/v{2}/{1}-v{2}-{3}.tar.gz",
+            "https://workers.cloudflare.com/get-wrangler-binary/{0}/{1}/v{2}/{3}.tar.gz",
```

Comment:
Are owner + tool_name validated on worker's side? I don't want that people start using our gateway for their own GitHub release

Reply:
They are validated on worker side.